app/controllers/events/view.js

```diff
@@ -6,6 +6,10 @@ export default Controller.extend({
       this.set('isEventDeleteModalOpen', true);
     },
     togglePublishState() {
+      if (this.get('model.event.location') === undefined) {
```

Comment:
Not sure whether we need to specifically check for undefined using ```===``` but I may be wrong.
@pradeepgangwar @mayank8318 Am i right ?

Reply:
@srv-twry it was wrong to compare to undefined. I have corrected and now checking if empty.
Can you please review again?